Person/persons.md

```diff
@@ -0,0 +1,386 @@
+    *attributes* | `Array[] : attributes` | The attribute details aggregated in an array
+    *deathdateEstimated* | Boolean | A true/false value to denote whether the deathDate is a correct estimate or not
+    *birthTime* | String | The time of birth of the person
+
+    ```
+    POST /person
+    {
+        "names": [
+            {
+            "givenName": "Mohit",
+            "familyName": "Kumar"
+            }
+        ],
+        "gender": "M",
+        "birthdate": "1997-09-02",
+        "addresses": [
+            {
+            "address1": "30, Vivekananda Layout, Munnekolal,Marathahalli",
+            "cityVillage": "Bengaluru",
+            "country": "India",
+            "postalCode": "560037"
+            }
+        ]
+    }
+    ```
+
+### Update a person
+
+* Update a target person with given UUID, this method only modifies properties in the request. Returns a `404 Not Found` 
+status if person not exists. If user not logged in to perform this action, a `401 Unauthorized` status returned. 
+
+    An example of the request is as follows : 
+
+    ```console
+    POST /person/:target_person_uuid
+    {
+        "age": 22,
+        "gender": "M",
+        "birthdate": "1997-01-13",
+    }
+    ```
+
+### Delete a person
+
+* Delete or Void a target person by its UUID. Returns a `404 Not Found` status if person not exists.If user 
+ not logged in to perform this action, a `401 Unauthorized` status returned.
+
+    #### Query Parameters
+
+    Parameter | Type | Description
+    --- | --- | ---
+    *purge* | `Boolean` | The resource will be voided unless purge = ‘true’
+
+    ```console
+    DELETE /person/:target_person_uuid?purge=true
+     ```
+
+
+### List person name subresource
+
+* List all the person name subresource corresponding to a `target_person_uuid`. Returns a `404 Not Found` status if person name not exists. If user not logged in to perform this action, a `401 unauthorized` status returned.
+
+```console
+GET /person/:target_person_uuid/name
+```
+
+* List all the person name subresource by it's `UUID` and corresponding to a `target_person_uuid`. Returns a `404 Not Found` status if person name not exists. If user not logged in to perform this action, a `401 unauthorized` status returned.
+
+```console
+GET /person/:target_person_uuid/name/:target_name_uuid
+```
+
+### Create person name subresource
+
+* To create a person name sub resource for a specific person resource you need to specify below properties in your request body.
+If user not logged in to perform this action, a `401 Unauthorized` status returned.
+
+#### Attributes
+
+Parameter | Type | Description
+--- | --- | ---
+*givenname* | `String` | name of the person
+*middlename* | `String` | middle name of the person
+*familyName* | `String` | family name/surname of the person
+*familyName2* | `String` | second family name/surname of the person
+*preferred* | `Boolean` | preferred/not preferred name
```

Comment:
`true` if this is the person's preferred name. When a person has only one name, it should be marked as preferred; when a person has multiple names, only one name can be preferred.